inst/rmarkdown/templates/fairness_report/skeleton/skeleton.Rmd

```diff
@@ -60,4 +109,6 @@ fairness_accuracy_tradeoff(resample_result)
 fairness_prediction_density(resample_result)
 ```
 
+## Interpretability
```

Comment:
The idea was, that you can use e.g. the `iml` package to automatically plot the feature importance and perhaps some feature effects for the user-provided model.

Reply:
Finished, do we need an example or do we need to do a full interpretability analysis?

Reply:
I would just do 2-3 plots

Reply:
I see.